tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -329,9 +329,15 @@ function read_cache_test_setup() {
     local enable_range_read_cache=$2
     local cache_ttl=$3
     local enable_parallel_downloads=$4
+    local disable_o_direct=$5
```

Comment:
Optional: one liner for this
`disable_o_direct=$([ -n "$5" ] && echo true || echo false)`